<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainPage</class>
 <widget class="QWidget" name="MainPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Main Page</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="btn_journal">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">
QPushButton {
	
	background-color: rgb(128, 60, 224);
	border-radius:10px;
    padding: 10px;
    border-radius: 10px;
	font: 10pt, Arial;
	color: white;

}

QPushButton:hover{
	
	background-color: rgb(193, 81, 241);
}</string>
     </property>
     <property name="text">
      <string>Journal</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_calender">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">
QPushButton {
	
	background-color: rgb(128, 60, 224);
	border-radius:10px;
    padding: 10px;
    border-radius: 10px;
	font: 10pt, Arial;
	color: white;

}

QPushButton:hover{
	
	background-color: rgb(193, 81, 241);
}</string>
     </property>
     <property name="text">
      <string>Calendar</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_moodtracker">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">
QPushButton {
	
	background-color: rgb(128, 60, 224);
	border-radius:10px;
    padding: 10px;
    border-radius: 10px;
	font: 10pt, Arial;
	color: white;

}

QPushButton:hover{
	
	background-color: rgb(193, 81, 241);
}</string>
     </property>
     <property name="text">
      <string>Mood Tracker</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="btn_logout">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: rgb(220, 20, 60);
    font: 14pt &quot;Arial&quot;;
    color: white;
    padding: 10px;
    border-radius: 10px;
}
QPushButton:hover {
    background-color: rgb(255, 69, 0);
}</string>
     </property>
     <property name="text">
      <string>Log Out</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
